scripts/train_dna2vec.py

```diff
@@ -52,7 +52,7 @@ def train(self, kmer_seq_generator):
 
     def write_vec(self):
         out_filename = '{}.w2v'.format(self.out_fileroot)
-        self.model.wv.save_word2vec_format(out_filename, binary=False)
+        self.model.wv.save_word2vec_format("."+out_filename, binary=False)
```

Comment:
Why do you prepend with `.`? This would make it a hidden file in linux.

Reply:
I must've not noticed that change as I was working on Windows. I'll change it in the latest push.